### DISCUSSION TITLE
G6 dropping connection

### DISCUSSION THREAD

**Discussion by TraceyEvers** [NightscoutFoundation/xDrip#2370] [Q&A]
Hi I have a motoroo g9 plus and recently started an Anubis g6 transmitter with xdrip. Moreso overnight, when the phone is beside my bed, but even during the day just less so, my xdrip alerts me that there has been missed readings and sure enough the number has a line through it. So far, I have had help to make sure all my aeting are correct, and the only fix I have found is turning Bluetooth off, closing the app, and restarting my phone. It then re-Starts the signal. Tonight it's list connection twice within the space of three hours. Yesterday it happened several times but overnight/early morning. Does anyone have any other suggestions? It seems to work mostly ok during the day. I have attached screenshots below and I have also disabled battery optimisation.

**Comment by TraceyEvers** [comment]
![Screenshot_20221004-131907](https://user-images.githubusercontent.com/45672270/194140956-08cdd132-de5e-4075-a62a-4ffc672ca430.png)
![Screenshot_20221004-131911](https://user-images.githubusercontent.com/45672270/194140970-b834d88c-a175-456d-a7f0-62a8b0c3f5bb.png)
![Screenshot_20221004-132132](https://user-images.githubusercontent.com/45672270/194140976-09e78d64-56e3-4c11-a0cb-b5921714f94c.png)
![Screenshot_20221004-132137](https://user-images.githubusercontent.com/45672270/194140980-626d0293-377e-40b0-9e4b-10d179e59653.png)
![Screenshot_20221004-133119](https://user-images.githubusercontent.com/45672270/194140983-f615a07c-69a2-4177-8738-d6ccc7357294.png)
![Screenshot_20221006-050152-594](https://user-images.githubusercontent.com/45672270/194141206-5f37ee68-f00a-4dcc-af7f-3b6efdc23b58.png)
![Screenshot_20221006-043404](https://user-images.githubusercontent.com/45672270/194149713-1f687f3f-1d0f-4e8a-920f-4271bf63cfa5.png)
![Screenshot_20221006-014413](https://user-images.githubusercontent.com/45672270/194149720-78dedf8a-c04a-4072-8c9f-c833c3f1a2cc.png)

**Reply by Navid200** [reply]
I would also add to your first post that you have disabled battery optimization of xDrip in Android settings.

**Reply by viq** [reply]
I've seen similar behaviour, with the additional information that version `e7be25a-2022.09.14` works fine, but `20220921` and `20221005` do exhibit this problem.
